### ISSUE TITLE
Default aruba.it sign rules

### ISSUE THREAD

**Issue by ale5000-git** [lieser/dkim_verifier#152]
I would like if possible to display, for example, mails from of @pec.it (signed by aruba.it) without the different domain warning as per default rules but without display errors on mail without DKIM signature.

**Comment by lieser** [comment]
For your own rules, simply use the [Rule type 2](https://github.com/lieser/dkim_verifier/wiki/Sign-rules#rule-type).

If a default rule is affected, please let me know, and I can change the rule type or remove it completely.

See also the [FAQ The add-on incorrectly says that en e-mail should be signed](https://github.com/lieser/dkim_verifier/wiki/FAQ#the-add-on-incorrectly-says-that-en-e-mail-should-be-signed).

---

Please clarify if the `@pec.it` was just an example, or if the default rule for that domain is actualy causing problems for you.

**Comment by ale5000-git** [comment]
It was only an example.

The default sign rules (the "Use default rules" option) hide the warning of different domains for the ones listed in the list but they are used only if "Check if e-mail should be signed" is enabled.
I would like to check if the SDID is correct as per default rules but doing nothing for mail witohut DKIM header.

**Comment by ale5000-git** [comment]
@lieser: In retrospect also the @pec.it rule could be the problem, I have checked all notifications and some have DKIM signature from aruba.it while some other doesn't have DKIM signature at all.

I can send you two samples if you need it.

**Comment by ale5000-git** [comment]
I have rechecked many mails and found this:
- `@pec.it` (some mails are signed with SDID aruba.it and some aren't signed at all)
- `@arubapec.it` (some mails are signed with SDID aruba.it and some aren't signed at all)
- `@pec.aruba.it` (some mails are signed with SDID aruba.it and some aren't signed at all)

- `@staff.aruba.it` (all mails are signed by aruba.it)
- `@aruba.it` (all mails are signed by aruba.it)

Only from PEC related addresses not all mails are signed.

**Comment by lieser** [comment]
Default rules changed in [2.1.0pre2](https://github.com/lieser/dkim_verifier/releases/tag/2.1.0pre2)

**Comment by ale5000-git** [comment]
Thanks, now I can easily distinguish real mails from fake ones.

**Comment by ale5000-git** [comment]
Unfortunately I have found another subdomain without DKIM, hopefully the latest one.
I know they are annoying with their inconsistency.

It is the subdomain that handle the "Mail Delivery System" notifications (the report for mails sent to unexistent addresses).
The mail attached in the body have the DKIM signature but the notification itself do not have it.

I got 2 mails from:
Mail Delivery System <`mail-daemon@smtpcmd15.ad.aruba.it`>
Mail Delivery System <`mail-daemon@smtpcmd02.ad.aruba.it`>

So `*@*.ad.aruba.it`
